draft-ietf-rats-architecture.md

```diff
@@ -1013,6 +1013,46 @@ an Attestation Result for any other purpose.
 
 # Security Considerations
 
+## Attester and Attestation Key Protection
+
+(this is a crude first pass to see what interest there is in this)
+
+Implementors need to pay close attention to the isolation and protection of the Attester and the factory processes for provisioning the Attestation Key Material. When these are compromised, the attestation becomes worthless because the attacker can forge attestations.
+
+Attestation applies to use cases with a range of security requirements, so the protections discussed here range from low to high security where low security  may be only application or process isolation by the devices operating system and high security involves specialized HW to defend against physical attacks on a chip.
+
+### On-Device Attester and Key Protection
+
+It is assumed that the Attester is an isolated computing environment like a process, a dedicated chip a TEE or such that collects the Claims, formats them and signs them with the Attestation Key. The Attester must be protected from modification to be sure it behaves correctly. There must also be confidentiality so that the signing key is not captured and used elsewhere to forge attestations.
```

Comment:
```suggestion
It is assumed that the Attester is located in an isolated environment of a device like a process, a dedicated chip a TEE or such that collects the Claims, formats them and signs them with an Attestation Key. The Attester must be protected from unauthorized modification to ensure it behaves correctly. There must also be confidentiality so that the signing key is not captured and used elsewhere to forge evidence.
```

Reply:
(did not address grammar issues as highlighted by Dave)